platform/ios/src/MGLMapView.h

```diff
@@ -668,12 +668,24 @@ MGL_EXPORT IB_DESIGNABLE
  Changing the value of this property updates the receiver immediately. If you
  want to animate the change, call `-setVisibleCoordinateBounds:animated:`
  instead.
+ 
+ To make the visible bounds go across the antimeridian or international date line,
+ specify some longitudes less than −180 degrees or greater than 180 degrees.
```

Comment:
This documentation is for a getter, not a setter, so it should talk about what it means to see a longitude beyond ±180°:

> If a longitude is less than −180 degrees or greater than 180 degrees, the visible bounds straddles the antimeridian or international date line.